### ISSUE TITLE
Need to use the H5P image library

### ISSUE THREAD

**Issue by tjb328** [h5p/h5p-memory-game#13]
The Memory Game does not use the H5P image library and are missing features such as ALT tags which are needed for accessibility.

**Comment by icc** [comment]
I agree, we should add improved readspeaker and keyboard support through ARIA as well. Thank you.
